lib/ansible/modules/network/netact/netact_cm_command.py

```diff
@@ -0,0 +1,364 @@
+            - CSV
+            - XLSX
+    fileName:
+        description:
+            - Specifies a file name.
+        required: false
+    inputFile:
+        description:
+            Specifies full path to plan file location for the import operation.
+            This parameter (inputFile) or the fileName parameter must be filled. If both are present then
+            the inputFile is used.
+        required: false
+    createBackupPlan:
+        description:
+            - Specifies if backup plan generation is enabled. Value can be set as true or false.
+        required: false
+    backupPlanName:
+        description:
+            - Specifies a backup plan name
+        required: false
+    extra_opts:
+        description:
+            Extra options to be set for operations. Check Configuration Management > Configuration Management
+            Operating Procedures > Command Line Operations in Nokia NetAct user documentation for further
+            information for extra options.
+        required: false
+
+author:
+    - Harri Tuominen (@hatuomin)
+'''
+
+EXAMPLES = '''
+# Pass in a message
+- name: Upload
+  netact_cm_command:
+    operation: "Upload"
+    opsname: 'Uploading_test'
+    dn: "PLMN-PLMN/MRBTS-746"
+    extra_opts: '-btsContentInUse true'
+
+- name: Provision
+  netact_cm_command:
+    operation: "Provision"
+    opsname: 'Provision_test'
+    dn: "PLMN-PLMN/MRBTS-746"
+    planName: 'mySiteTemplate'
+    type: 'actual'
+    createBackupPlan: true
+    backupPlanName: 'myBackupPlanName'
+
+- name: Export and fetching data from target
+  netact_cm_command:
+    operation: "Export"
+    opsname: 'Export_test'
+    planName: 'mySiteTemplate'
+    type: 'actual'
+    fileName: 'exportTest.xml'
+- fetch:
+    src: /var/opt/nokia/oss/global/racops/export/exportTest.xml
+    dest: fetched
+
+- name: Import
+  netact_cm_command:
+    operation: "Import"
+    opsname: 'Import_test'
+    fileFormat: 'CSV'
+    type: 'plan'
+    fileName: 'myCSVFile'
+    planName: 'myPlanName'
+    extra_ops: 'enablePolicyPlans true'
+
+# fail the module
+- name: Test failure of the module
+  netact_cm_command:
+    name: fail me
+'''
+
+RETURN = '''
+original_message:
+    description: The original name param that was passed in
+    returned: Command line
+    type: string
+    sample: '/opt/oss/bin/racclimx.sh -op Upload -opsName Uploading_testi -DN PLMN-PLMN/MRBTS-746'
+message:
+    description: The output message that the netact_cm_command module generates
+    returned: Command output message
+    type: string
+changed:
+    description: data changed
+    returned: true if data is changed
+    type: boolean
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+racclimx = '/opt/oss/bin/racclimx.sh'
+
+
+def run_module():
+    """
+    Main module where option are handled and command is executed
+    :return:
+    """
+    # define the available arguments/parameters that a user can pass to
+    # the module
+    module_args = dict(
```

Comment:
There are various `aliases` listed here, sometimes just the upper case version. Not sure if they add much value.

Reply:
I will clean this aliases